numpy/polynomial/chebyshev.py

```diff
@@ -1807,11 +1947,19 @@ def chebinterpolate(func, deg, args=()):
 
     Examples
     --------
+    >>> import numpy as np
```

Comment:
here and elsewhere I think you can treat the `import numpy as np` as implicit; I think it is done by convention and for concision in docstrings